app/code/core/Mage/Adminhtml/Block/Widget/Tabs.php

```diff
@@ -189,6 +189,7 @@ protected function _setActiveTab($tabId)
 
     protected function _beforeToHtml()
     {
+        Mage::dispatchEvent('adminhtml_widget_tabs_add_after', array('block' => $this));
```

Comment:
I'd suggest either a rename for consistency:

```suggestion
        Mage::dispatchEvent('adminhtml_widget_tabs_html_before', array('block' => $this));
```

Or, adding a new method to dispatch the event after the layout is prepared which is always where the default tabs are added:

```php
    protected function _prepareLayout()
    {
        parent::_prepareLayout();
        Mage::dispatchEvent('adminhtml_widget_tabs_prepare_layout_after', array('block' => $this));
        return $this;
    }
```